src/md/enc/metamodelrw.cpp

```diff
@@ -8780,7 +8780,7 @@ bool FilterTable::IsTokenMarked(
         //
         return true;
     }
-    return ( (*Get(rid)) & bitMarked ? true : false);
+    return ( ((*Get(rid)) & bitMarked) ? true : false);
```

Comment:
@janvorli Sure, but I'm kind of worried by DNFBOT and the legal stuff. Why was my first PR accepted without requiring CLA but this one does require CLA?